### PR TITLE
Add support for skipping occurrence

### PR DESCRIPTION
**Pull request by SCdF**

### CODE REVIEW

File: code/multiple_cursors.py [hunk 1]
```diff
@@ -24,6 +24,9 @@ def multi_cursor_select_fewer_occurrences():
     def multi_cursor_select_more_occurrences():
         """Adds cursor at next occurrence of selection"""
 
+    def multi_cursor_select_skip_occurrences():
```
Comment (pokey) [inline code/multiple_cursors.py:27]: I find this action name a bit confusing.  How about `multi_cursor_skip_occurrence` instead?

**Review by pokey — COMMENTED** [review]
this looks good to me other than renaming action for clarity

File: code/multiple_cursors.py (hunk 1 above)
Comment (SCdF) [reply code/multiple_cursors.py:27]: okey dokey

**Comment by SCdF** [comment]
Function name change

**Review by pokey — APPROVED** [review]

**Pull request merged**